assembl/graphql/idea.py

```diff
@@ -305,7 +305,8 @@ def resolve_posts(self, args, context, info):
 
         query = models.Post.query.join(
             related, models.Post.id == related.c.post_id
-        )
+        ).options(joinedload(models.Post.creator))
```

Comment:
You added this because you added creator on the IdeaWithPosts query, but you shouldn't touch IdeaWithPosts query, this will impact the perf of the query for the tree in thread phase. You may create a new query for your case.
In this case, add the joinedload conditionally, like I did for sentimentsCounts:
`if 'creator' in fields.get('edges', {}).get('node', {}):`

Reply:
Ok, that will be more efficient indeed. Thank you!